recipes/ckanweb-deploy-exts.rb

```diff
@@ -383,12 +383,18 @@
 
 	if "#{pluginname}".eql? 'clamav'
 		if not batchnode
-			package 'clamav'
-			package 'clamd'
+			package 'clam' do
+				package_name ['clamav', 'clamav-update', 'clamd']
+			end
 
 			bash "Enable Clam daemons" do
 				code <<-EOS
 					freshclam
+					# Default clamd config doesn't enable any socket and will therefore fail
```

Comment:
For the time being, lets only run this in ckan test (dev). not in our normal system just yet. 

It would be best to move from inline to batch job but that would need more uplift time.

Reply:
Yes, the plugin is only enabled on test-dev.data.qld.gov.au